features/admin.server-configurations.v1/pages/connector-edit-page.tsx

```diff
@@ -575,7 +596,7 @@ export const ConnectorEditPage: FunctionComponent<ConnectorEditPageInterface> =
             case ServerConfigurationsConstants.ACCOUNT_RECOVERY_CONNECTOR_ID:
```

Comment:
can remove this complete case since the toggle is not needed

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/5808/commits/64392b4a07873e3e152ce8db7d01f0c702462640

Reply:
removed